src/bank.ts

```diff
@@ -141,4 +141,12 @@ export class Bank extends ServerNode {
   async getValidators(options: Partial<PaginationOptions> = {}) {
     return await super.getPaginatedData<PaginatedValidatorEntry>("/validators", options);
   }
+
+  /**
+   * Gets the PrimaryValidator for the current bank.
+   */
+  async getBankPV() {
+    const { primary_validator } = await await this.getConfig();
```

Comment:
this looks like a new way to write awaits, does it work?

Reply:
Haha I accidentally wrote await twice, my bad 😅. Not sure how I missed that